staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -227,6 +228,7 @@ func describerMap(clientConfig *rest.Config) (map[schema.GroupKind]ResourceDescr
 		{Group: certificatesv1beta1.GroupName, Kind: "CertificateSigningRequest"}: &CertificateSigningRequestDescriber{c},
 		{Group: storagev1.GroupName, Kind: "StorageClass"}:                        &StorageClassDescriber{c},
 		{Group: storagev1.GroupName, Kind: "CSINode"}:                             &CSINodeDescriber{c},
+		{Group: storagev1alpha1.GroupName, Kind: "VolumeAttributesClass"}:         &VolumeAttributesClassDescriber{c},
```

Comment:
Could you please add more information about this resource in the PR description such as it's status(alpha), KEP, etc.

Reply:
ah, you have already added the KEP in release note https://github.com/kubernetes/enhancements/issues/3751